### ISSUE TITLE
Support image-captions

### ISSUE THREAD

**Issue by eibensteiner** [eibensteiner/eibensteiner.com#5]
It should be possible to add captions to images, not only for accessibility reasons but also for a better understanding of the context an image is placed in.
